File: library/src/main/java/com/goldenpie/devs/pincodeview/core/Listeners.java
package com.goldenpie.devs.pincodeview.core;

public class Listeners {
    public interface PinEnteredListener {
        void onPinEntered(String pinCode);
    }

    public interface PinMismatchListener {
        void onPinMismatch();
    }

    public interface PinReEnterListener {
        void onPinReEnterStarted();
    }
}

File: library/src/main/java/com/goldenpie/devs/pincodeview/core/Listeners.kt
package com.goldenpie.devs.pincodeview.core

open class Listeners {
    interface PinEnteredListener {
        fun onPinEntered(pinCode: String?)
    }

    interface PinMismatchListener {
        fun onPinMismatch()
    }

    interface PinReEnterListener {
        fun onPinReEnterStarted()
    }
}